lib/node_modules/@stdlib/ndarray/base/any-by/lib/2d.js

```diff
@@ -0,0 +1,136 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var strides2order = require( '@stdlib/ndarray/base/strides2order' );
+var zeroTo = require( '@stdlib/array/base/zero-to' );
+var reverse = require( '@stdlib/array/base/reverse' );
+var take = require( '@stdlib/array/base/take-indexed' );
+
+
+// MAIN //
+
+/**
+* Tests whether at least one element in an ndarray pass a test implemented by a predicate function.
```

Comment:
```suggestion
* Tests whether at least one element in an ndarray passes a test implemented by a predicate function.
```

This applies here and everywhere.